charts/kuberentes-dashboard.yaml

```diff
@@ -0,0 +1,440 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
Note: name of the file contains a typo

Idea: generate it regularly from the Helm Chart with default values.

Reply:
I was thinking about that but I am not sure if we can generate it reliably without custom post-processing.